### PR TITLE
feature/cover-service-with-it

### PR DESCRIPTION
**Pull request by valentinashaban**

### CODE REVIEW

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+    @Autowired
+    private MoneyTransferService moneyTransferService;
+
+    private User user;
+    private MoneyTransfer moneyTransfer;
+    private Wherefrom wherefrom;
+
+    @Before
+    public void init() {
```
Comment (dcadea) [inline src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: init -> setUp

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+        assertEquals(Arrays.asList(persistedMoneyTransfer), requiredMoneyTransfer);
+    }
+
+    @Test
+    public void testGetMoneyTransferLastWeek() {
+        userDao.persist(user);
+        wherefromDao.persist(wherefrom);
+
+        MoneyTransfer newMT = MoneyTransfer.builder()
```
Comment (dcadea) [inline src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: externalize this builder.
add a private method:

private MoneyTransfer createNewMoneyTransfer(User user, Wherefrom wherefrom) {
return MoneyTransfer.builder().....
}

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+                                           .withAmount(52.0)
```
Comment (dcadea) [inline src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: create local variable AMOUNT = 52.0

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+                                           .withDate(new Date(System.currentTimeMillis() - 172800000L))
```
Comment (dcadea) [inline src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: create local variable for this date

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java (hunk 1 above)
Comment (valentinashaban) [reply src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: DONE

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java (hunk 3 above)
Comment (valentinashaban) [reply src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: DONE

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java (hunk 4 above)
Comment (valentinashaban) [reply src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: DONE

File: src/test/java/com/endava/service/MoneyTransferServiceImplIT.java (hunk 2 above)
Comment (valentinashaban) [reply src/test/java/com/endava/service/MoneyTransferServiceImplIT.java]: DONE

**Pull request merged**